src/additional-components/Controls/index.tsx

```diff
@@ -73,8 +81,28 @@ const Controls: FC<ControlProps> = ({
 
   useEffect(() => {
     setIsVisible(true);
+    setInteractive?.(true);
   }, []);
 
+  // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values
+  const isControlPressed = useKeyPress("Control")
+  const isCommandPressed = useKeyPress("Meta")
+  const isValidControlOrCommandPressPerOS = (OS === OSName.Windows && isControlPressed) || (OS === OSName.Mac && isCommandPressed)
+  const isPlusPressed = useKeyPress("=")
+  const isMinusPressed = useKeyPress("-")
+  const isZoomingIn = isValidControlOrCommandPressPerOS && isPlusPressed
+  const isZoomingOut = isValidControlOrCommandPressPerOS && isMinusPressed
+
+  useEffect(() => {
+    if (isZoomingIn) {
+      onZoomInHandler?.();
+    }
+
+    if (isZoomingOut) {
+      onZoomOutHandler?.();
+    }
+  }, [onZoomInHandler, onZoomOutHandler, isZoomingIn, isZoomingOut]);
```

Comment:
So this part is for the keyboard access right? I think we should expose a way to trigger zoom from outside the library if there isn't one already, so that we can move this logic there. Alternately, we could expose a way to pass key combinations to the library and have the library handle the actual event binding, but the specific keys used (including ctrl/meta modifiers) constitute Creator business logic and should be in Creator.